run/GCHP/HISTORY.rc.templates/HISTORY.rc.fullchem

```diff
@@ -1610,7 +1270,10 @@ COLLECTIONS: @#'DefaultCollection',
 #
 # UV fluxes (from the photolysis module)
 #
-# Available for all full-chemistry simulations with photolysis
+# Available for all full-chemistry simulations with photolysis\
+#
+# NOTE: There seems to be an issue with this diagnostic in GCHP.
```

Comment:
Is this still the case after the all diags integration test fixes?

Reply:
Yes, this diagnostic causes a seg fault.  @lizziel knows about this but since this is a low-priority item we will fix it later.

Reply:
Thanks for confirming.